Description: Free escaped strings with curl_free instead of g_free
Author: Vincent Legout <vlegout@debian.org>
Last-Update: 2012-03-23

Index: gimmix/src/gimmix-lyrics.c
===================================================================
--- gimmix.orig/src/gimmix-lyrics.c	2012-03-23 20:52:34.081280063 +0100
+++ gimmix/src/gimmix-lyrics.c	2012-03-23 20:54:54.925279308 +0100
@@ -316,8 +316,8 @@
 		char *artist_e = lyrics_url_encode (search_artist);
 		char *title_e = lyrics_url_encode (search_title);
 		url = g_strdup_printf ("%s&artist=%s&songtitle=%s", SEARCH_URL, artist_e, title_e);
-		g_free (artist_e);
-		g_free (title_e);
+		curl_free (artist_e);
+		curl_free (title_e);
 		//g_print ("%s\n", url);
 		ret = lyrics_perform_search (url);
 		g_free (url);
Index: gimmix/src/gimmix-covers.c
===================================================================
--- gimmix.orig/src/gimmix-covers.c	2012-03-23 20:52:34.081280063 +0100
+++ gimmix/src/gimmix-covers.c	2012-03-23 20:57:28.825280835 +0100
@@ -609,8 +609,8 @@
 	
 	g_free (old_path);
 	g_free (new_path);
-	g_free (artist_e);
-	g_free (album_e);
+	curl_free (artist_e);
+	curl_free (album_e);
 	//g_free (key);
 
 	return;
@@ -640,8 +640,8 @@
 	}
 	g_free (temp);
 	g_free (path);
-	g_free (artist_e);
-	g_free (album_e);
+	curl_free (artist_e);
+	curl_free (album_e);
 	
 	return;
 }
@@ -670,8 +670,8 @@
 	album_e = gimmix_url_encode (s->album);
 	temp = cfg_get_path_to_config_file (COVERS_DIR);
 	path = g_strdup_printf ("%s/%s-%s.albuminfo", temp, artist_e, album_e);
-	g_free (artist_e);
-	g_free (album_e);
+	curl_free (artist_e);
+	curl_free (album_e);
 	
 	if ((fp=fopen(path,"r")))
 	{
@@ -695,8 +695,8 @@
 	path = g_strdup_printf ("%s/%s-%s.albuminfo", p, artist_e, album_e);
 	g_free (p);
 	g_free (temp);
-	g_free (artist_e);
-	g_free (album_e);
+	curl_free (artist_e);
+	curl_free (album_e);
 	
 	if ((fp=fopen(path,"r")))
 	{
